prepare_confidential_upload.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# this takes the confidential data from the appropriate places in our build directories and puts it all into one folder for uploading to Google Drive
+# warning: do not commit this data accidentally!
```

Comment:
Took me a couple re-reads to get this, should probably explicitly say that the folder/zip file shouldn't be committed as opposed this script.

Reply:
Oh, I just copied and slightly modified it from the expand script